.circleci/config.yml

```diff
@@ -492,12 +492,11 @@ jobs:
       - run:
           name: Docker build
           command: |
-            ROUTER_RELEASE=${VERSION:1}
```

Comment:
This was removing the letter `v` from the Git release tag (which is inclusive of the `v`) so that it could pass it to `ROUTER_RELEASE` below.  The Docker image build now defaults to `latest` (and accepts `--build-arg ROUTER_RELEASE=latest`), so we can pass this value in opaquely now.

This does mean that users do pass `--build-arg ROUTER_RELEASE=...` they need to pass `ROUTER_RELEASE=v0.16.0` or `ROUTER_RELEASE=latest`, but that actually seems intuitive to me in the context of the notion of "release".

It is a breaking change for current Router image builders, but this seems like a good time to make a breaking change of that sort!